src/gateways/googleAnalyticsClient.spec.ts

```diff
@@ -122,6 +123,20 @@ describe('GoogleAnalyticsClient', () => {
 
         expect(client.getPendingCommands().length).toBe(0);
       });
+
+      it('doens\'t send and flush the commands if trackingIsDisabled is true', async() => {
+        jest.spyOn(analytics, 'trackingIsDisabled').mockReturnValueOnce(true);
+
+        client.trackPageView('/some/path');
+        expect(mockGa).not.toHaveBeenCalled();
+        expect(client.getPendingCommands().length).toBe(1);
+
+        client.setTrackingIds(['foo']);
+        expect(mockGa).not.toHaveBeenCalledWith('tfoo.set', 'queueTime', expect.any(Number));
+        expect(mockGa).not.toHaveBeenCalledWith('tfoo.send', {hitType: 'pageview', page: '/some/path'});
```

Comment:
maybe change this to `expect(mockGa).not.toHaveBeenCalled()` to be sure that it is really not called cuz this test will pass if someone mistypes the name

Reply:
same for the other test